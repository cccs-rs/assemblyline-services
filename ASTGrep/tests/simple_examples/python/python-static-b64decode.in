import base64
from base64 import b64decode

base64.b64decode("YWJj")
b64decode('YWJj')

cmd = "YWJj"
base64.b64decode(cmd)
subprocess.Popen([pycmd, '-c', base64.b64decode(cmd).decode()])
