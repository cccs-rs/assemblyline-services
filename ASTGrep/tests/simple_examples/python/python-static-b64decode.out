import base64
from base64 import b64decode

b'abc'
b'abc'

cmd = "YWJj"
b'abc'
subprocess.Popen([pycmd, '-c', 'abc'])
